common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Random;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import com.google.common.collect.Lists;
```

Comment:
Nit: I wouldn't import this library just to make an ArrayList. In Java 8, just: `List<ByteBuf> retained = new ArrayList<>();` which is no more complex.

Reply:
Ok, I will update.